patches/common/chromium/accelerator.patch

```diff
@@ -19,17 +19,17 @@ index 7e55ef366ac8320f730cdcb268453b1fa2710887..c3fb98b426cd7c12f66eaaf358f4ff18
 @@ -21,9 +22,7 @@
  #include <windows.h>
  #endif
- 
+
```

Comment:
those spaces are important, leave them in please! this patch should be exactly as exported by `git-export-patches`.

Reply:
I knew there was something up with those spaces, gonna fix asap